js/client-rooms.js

```diff
@@ -24,6 +24,12 @@
 			app.send('/cmd rooms');
 			app.user.on('change:named', this.updateUser, this);
 			this.update();
+			this.chatroomInterval = setInterval(() => {
+				if (app.curSideRoom?.id === 'rooms') {
```

Comment:
```suggestion
			this.chatroomInterval = setInterval(function () {
				if (app.curSideRoom && app.curSideRoom.id === 'rooms') {
```

Reply:
i hate internet explorer users

Reply:
Yea I caught it as you commented. Can't we use babel? It would automatically convert our code to something that would work with es3

Reply:
That would be something to discuss with @Zarel. I don't know what the plan is for the preact rewrite.

Reply:
Ok, I'll see if I can reach out to him again

Reply:
Someone may need to rerun the workflow in the mean time. I can't trigger it because I'm a first time contributor

Reply:
Zarel is pretty busy right now, I'll ask him to consider using a transpiler for the client when he's finished with some more important tasks.

Reply:
Thanks! I remember talking to him a year or two about helping out with the Preact rewrite. I be happy to contribute if he's still working on it